Draw.it.Server/Hubs/GameplayHub.cs

```diff
@@ -41,19 +41,23 @@ public override async Task OnConnectedAsync()
         var playerStatuses = GetPlayerStatuses(roomId);
         await Clients.Group(roomId).SendAsync("ReceivePlayerStatuses", playerStatuses);
 
+        var strokes = _gameService.GetCanvasStrokes(roomId);
+        if (strokes != null && strokes.Any())
+        {
+            await Clients.Caller.SendAsync("ReceiveCanvasState", strokes);
+        }
```

Comment:
```suggestion
        await Clients.Caller.SendAsync("ReceiveCanvasState", game.CanvasStrokes);
```

I suggest you get rid of unnecessary checks here, since:

<img width="769" height="212" alt="Image" src="https://github.com/user-attachments/assets/90f6622b-a56d-4237-99c2-db1a6a28c93d" />

Also, `GetCanvasStrokes` method has no real use, because you can get the canvas by simply `game.CanvasStrokes`. So, better remove it to not overcomplicate things